### ISSUE TITLE
[bug]: torch 2.1.0 hash does not match when installing from pip with python 3.11.5

### ISSUE THREAD

**Issue by Rucadi** [invoke-ai/InvokeAI#5006]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Linux

### GPU

cuda

### VRAM

8GB

### What version did you experience this issue on?

latest (pip install)

### What happened?

  × pip subprocess to install backend dependencies did not run successfully.
  │ exit code: 1
  ╰─> [11 lines of output]
      Looking in indexes: https://pypi.org/simple, https://download.pytorch.org/whl/cu118
      Collecting numpy
        Using cached numpy-1.26.1-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (61 kB)
      Collecting torch
        Downloading https://download.pytorch.org/whl/cu118/torch-2.1.0%2Bcu118-cp311-cp311-linux_x86_64.whl (2325.9 MB)
           ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 GB 4.8 MB/s eta 0:00:00
      ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
          torch from https://download.pytorch.org/whl/cu118/torch-2.1.0%2Bcu118-cp311-cp311-linux_x86_64.whl#sha256=bcb17e2de6ca634d326203694d0bfb552587335e536c1917be3f28c5664b5506:
              Expected sha256 bcb17e2de6ca634d326203694d0bfb552587335e536c1917be3f28c5664b5506
                   Got        e5cc53ec4db3a003c8b73738adf4ab019369ab17e929261921d303ae4e7426e2
      
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.

**Comment by Millu** [comment]
I don't think this is an InvokeAI issue - we don't have any hashes in the requirements file
